src/background.ts

```diff
@@ -52,6 +53,17 @@ class ShortExt {
             });
     }
 
+    isEmptyTab(tab: chrome.tabs.Tab) {
+        return !tab.hasOwnProperty("url");
+    }
+
+    onIconClick = (webUi: string) => {
+        chrome.browserAction.onClicked.addListener(tab => {
+            if(this.isEmptyTab(tab)) {
+                chrome.tabs.create({ url: `${webUi}/?long_link=${tab.url}` });
```

Comment:
```suggestion
                     return;
                 }
                chrome.tabs.create({ url: `${webUi}/?long_link=${tab.url}` });
```

Reply:
Please compare the conditional inside the `if` statement with the previous comments.

Reply:
oops. yes